CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## **[Unreleased]**
 
+- [#1212](https://github.com/wasmerio/wasmer/pull/1212) [BREAKING CHANGE] Add `--generate-debug-info` and `-g` flags to `wasmer run` to generate debug information during compilation that is passed via the GDB JIT interface to a debugger to allow source-level debugging of Wasm files. Currently only available on clif-backend. As part of this change, the public middleware API is broken, there is now a `source_loc` parameter that should be passed through if applicable. Additionally there are breaking changes on the compiler trait functions such as `feed_local`, `feed_event` as well as `ModuleCodeGenerator::finalize`.
```

Comment:
```suggestion
- [#1212](https://github.com/wasmerio/wasmer/pull/1212) [BREAKING CHANGE on internal APIs] Add `--generate-debug-info` and `-g` flags to `wasmer run` to generate debug information during compilation that is passed via the GDB JIT interface to a debugger to allow source-level debugging of Wasm files. Currently only available on clif-backend. As part of this change, the public middleware API has a breaking change, there is now a `source_loc` parameter that should be passed through if applicable. Additionally there are breaking changes on the compiler trait functions such as `feed_local`, `feed_event` as well as `ModuleCodeGenerator::finalize`.
```

Reply:
> BREAKING CHANGE on internal APIs

Do you mean external APIs?

Reply:
I wanted to remove some alarmism from it.
We can do `BREAKING CHANGE on Middleware and internal APIs`